applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,206 @@
+//'use strict';
+'require view';
+'require form';
+
+return view.extend({
+	render: function() {
+	var m, s, o;
+
+	m = new form.Map('usteer', _('usteer'), _('usteer Configuration.'));
+
+	s = m.section(form.TypedSection, 'usteer', _('options'), _('First four options are mandatory. Also be sure to enable rrm reports, 80211kvr, etc see: https://openwrt.org/docs/guide-user/network/wifi/usteer'));
+	s.anonymous = true;
+
+	s.option(form.Value, 'network', _('network'), _('The network interface for inter-AP communication)'));
+
+	o = s.option(form.Flag, 'syslog', _('Syslog'), _('Log messages to syslog (0/1)'));
+	o.default = '1';
+	o.rmempty = false;
+
+	o = s.option(form.Flag, 'ipv6', _('ipv6 mode'), _('Use IPv6 for remote exchange'));
+	o.rmempty = false;
+
+	o = s.option(form.ListValue, 'debug_level', _('debug level'), _('Debug level'));
+	o.placeholder = 'lan';
+	o.value('0','0 Fatal');
+	o.value('1','1 info');
+	o.value('2','2 Verbose');
+	o.value('3','3 Some debug');
+	o.value('4','4 network packet info');
+	o.value('5','5 all debug messages');
+	o.rmempty = false;
+	o.editable = true;
+
+	o = s.option(form.Value, 'max_neighbour_reports', _('max neighbour reports'), _('Maximum number of neighbor reports set for a node'));
```

Comment:
Please capitalize option titles, like it is done elsewhere in LuCI.

Reply:
done